utils/keeper-data-dumper/main.cpp

```diff
@@ -15,7 +15,7 @@ using namespace DB;
 
 void dumpMachine(std::shared_ptr<KeeperStateMachine> machine)
 {
-    auto & storage = machine->getStorage();
+    auto & storage = machine->getStorageForUnitTests();
```

Comment:
Please revert the name to `getStorage` because it looks a bit strange calling `ForUnitTests` when it's not unit tests.

Reply:
Ok, but we need sort of a note this is not for basic usage as it violates invariants (or we need a lock in this method).
I'll change to the latter.

Reply:
What do you think about renaming to getStorageUnsafe?
We can't really use TSA as we return a reference without the ability to attach a lock to it